apps/mobile/src/components/page/page.layout.tsx

```diff
@@ -1,63 +1,32 @@
-import { useCallback, useState } from 'react';
 import { RefreshControl } from 'react-native';
 import { ScrollView } from 'react-native-gesture-handler';
 import { useSafeAreaInsets } from 'react-native-safe-area-context';
 
 import { useActionBarOffset } from '@/components/action-bar/action-bar';
-import { queryClient } from '@/queries/query';
+import { useRefreshHandler } from '@/hooks/use-refresh-handler';
 
-import { HttpCacheService } from '@leather.io/services';
 import { Box, HasChildren } from '@leather.io/ui/native';
 
-export function useRefreshHandler() {
-  const [refreshing, setRefreshing] = useState(false);
-
-  const onRefresh = useCallback(() => {
-    setRefreshing(true);
-    void queryClient.invalidateQueries({
-      predicate: query =>
-        !(query.queryKey[0] as string).startsWith(HttpCacheService.CACHE_KEY_PREFIX),
-    });
-    setTimeout(() => {
-      setRefreshing(false);
-    }, 2000);
-  }, []);
-
-  return { refreshing, onRefresh };
-}
-
-function ScrollableContent({ children }: HasChildren) {
+export function PageLayout({ children }: HasChildren) {
   const { actionBarOffset } = useActionBarOffset();
   const { top } = useSafeAreaInsets();
   const contentOffsetTop = top;
 
   const { refreshing, onRefresh } = useRefreshHandler();
-
-  return (
-    <ScrollView
-      contentContainerStyle={{ paddingBottom: actionBarOffset }}
-      refreshControl={
-        <RefreshControl
-          refreshing={refreshing}
-          onRefresh={onRefresh}
-          progressViewOffset={contentOffsetTop}
-        />
-      }
-    >
-      {children}
-    </ScrollView>
-  );
-}
-
-interface PageLayoutProps extends HasChildren {
-  scrollable?: boolean;
-}
-export function PageLayout({ children, scrollable = true }: PageLayoutProps) {
   return (
-    <>
-      <Box flex={1} bg="ink.background-primary">
-        {scrollable ? <ScrollableContent>{children}</ScrollableContent> : children}
-      </Box>
-    </>
+    <Box flex={1} bg="ink.background-primary">
```

Comment:
Previously, I made `PageLayout` so we could have a wrapper for places we needed to be able to drag and refresh to invalidate queries. 

I added a flag of `scrollable` to wrap it in `ScrollView` or not as during early stages to `FlashList` implementation it gave warnings about using it inside of a scroll area.

It turns out that not having it in a scroll area was the main cause of this bug with disappearing content - LEA-2660. 

@tigranpetrossian is [working on a refactor](https://trustmachines.slack.com/archives/C05LAP952E7/p1749800036550979) of this setup to instead use a `Screen` component. 

For now I've:
- added `FlashList` for `Activity` inside of `ScrollView`
- refactored out the hook for `useRefreshHandler`